miasm2/jitter/llvmconvert.py

```diff
@@ -398,7 +398,7 @@ def memory_write(self, func, addr, size, value):
             builder.store(value, ret)
             value_ptr = builder.bitcast(ret, llvm_ir.IntType(8).as_pointer())
 
-            ret = builder.call(
+            _ = builder.call(
```

Comment:
As you do in this same commit but for another file, the returned value can be ignored, so the `_ = ` be removed (but I note the Rust-ism :smile: )